hack/overrides/jaeger.yml

```diff
@@ -20,6 +22,9 @@ services:
       - --log-level=debug
     environment:
       COLLECTOR_ZIPKIN_HTTP_PORT: 9411
+      SPAN_STORAGE_TYPE: elasticsearch
+      ES_SERVER_URLS: http://elasticsearch:9200
+      ES_TAGS_AS_FIELDS_ALL: "true"
```

Comment:
should we include the change to number of replicas like how we did for hush-house? https://github.com/concourse/hush-house/commit/5b1e70245e540f166222eba12dba2ec732aec590

Reply:
yup, don't want those yellow indices
<img width="1109" alt="Screen Shot 2020-04-17 at 9 39 05 AM" src="https://user-images.githubusercontent.com/22056320/79575175-61efb700-808f-11ea-8d1f-49cc29b14576.png">